apps/opik-backend/config.yml

```diff
@@ -69,7 +69,14 @@ server:
     enabled: true
 
 rateLimit:
-    enabled: ${RATE_LIMIT_ENABLED:-false}
-    generalLimit:
-      limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
-      durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+  enabled: ${RATE_LIMIT_ENABLED:-false}
+  generalLimit:
+    limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-10000}
+    durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-60}
+
+usageReport:
+  enabled: ${OPIK_USAGE_REPORT_ENABLED:-true}
+  url: ${OPIK_USAGE_REPORT_URL:-https://stats.comet.com/notify/event/}
+
+metadata:
+  version: ${OPIK_VERSION:-latest}
```

Comment:
@thiagohora latest version by default is misleading, can we get the version from the docker image itself ? how will this work if you 
* git clone the repo
* run docker compose
* server is up

what version will be sent?

Reply:
Running via docker, there is an argument that becomes an Envvar, so in this case, when cloning the repo, the version would be `latest`. 

![Screenshot 2024-09-30 at 10 23 10](https://github.com/user-attachments/assets/f021728d-6fec-40ff-b722-e75f07198868)

Reply:
However, to solve this, we would need some templates and hardcode the version in the docker file.